client/src/features/common/Images.tsx

```diff
@@ -11,7 +11,7 @@ import IiifImageViewer from './IiifImageViewer'
 import WikiDataImageViewer from './WikiDataImageViewer'
 
 const Images: React.FC<{ entity: IEntity }> = ({ entity }) => {
-  const work = new WorkParser(entity)
+  const work = new EntityParser(entity)
```

Comment:
I'm wondering if we should call this variable `objectOrWork` since it could be either one?

Reply:
@clarkepeterf I changed it to `element` as that is usually my go-to variable name if I'm using EntityParser. That or `entity` but `entity` is already being used.